src/test/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategyFactoryTest.java

```diff
@@ -73,6 +74,7 @@ public void checkProperties() {
     public void createDiscoveryStrategy() {
         HashMap<String, Comparable> properties = new HashMap<String, Comparable>();
         properties.put(KubernetesProperties.KUBERNETES_API_TOKEN.key(), API_TOKEN);
+        properties.put(KubernetesProperties.KUBERNETES_CA_CERTIFICATE.key(), CA_CERTIFICATE);
```

Comment:
Hmm...the certificate is different for each cluster. So, when you create a new k8 cluster, then kubernetes master generates CA certificate. In k8 Kubernetes Master is also a Certificate Authority. Then, we cannot create a certificate that would always work.

Reply:
It can be a fake certificate but just showing how to configure real life case such as how to embed a certificate into hazelcast-client.xml or programmatically configure...

a complete hazelcast.xml with a sample like below.

```
<hazelcast-client xmlns="http://www.hazelcast.com/schema/client-config"
                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                  xsi:schemaLocation="http://www.hazelcast.com/schema/client-config
                  http://www.hazelcast.com/schema/client-config/hazelcast-client-config-3.11.xsd">
    <network>
        <kubernetes enabled="true">
            <api-token>eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkZWZhdWx0Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9zZWNyZXQubmFtZSI6ImRlZmF1bHQtdG9rZW4taHhjMnciLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoiZGVmYXVsdCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjFkODM3MTc4LTFlNmMtMTFlOS05NGZjLTQyMDEwYThlMDA4YiIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkZWZhdWx0OmRlZmF1bHQifQ.QxlpQS_ziqV91RSJHc-gRonQPh5-V_qCoOWBPACnqfi0ihnor1kXSwJ_BcPNRIy3G8sA0GMNJgs6VPSmwW5Ia9Cuz-zJmmTNlXl5THULtBLlSZeMgh5mTCuaTeTKQjdL2QEoM1WfDeE6ww1M7Fkai8eHrDVD0oQQyCyNtWzWfTHXeNq9ra0bjgyONAiwm7N8_mdS9au3hBnqdIaVlF7j4wyYG7OZQzYs_HkPVYhCTpoICMRHMjww2AvSi1Hp6w11_vq7QVSHWGak1SM0hVstNGWEGt1lfdBqz7kywQ1etruocoYqePjNsBJLLZ_kfJ4EmwqDzcuS7XeuYsV3kF13Dw</api-token>
             <ca-certificate>
                -----BEGIN CERTIFICATE-----
                MIIDDDCCAfSgAwIBAgIRAKpKYHUsJhUGs6YkNWrGc/UwDQYJKoZIhvcNAQELBQAw
                LzEtMCsGA1UEAxMkOWJkZmQ2ZTItMTJmZC00OWFmLWFmZDgtMWY0ZWUyNGEyYWM5
                MB4XDTE5MDEyMjE2MzMyMFoXDTI0MDEyMTE3MzMyMFowLzEtMCsGA1UEAxMkOWJk
                ZmQ2ZTItMTJmZC00OWFmLWFmZDgtMWY0ZWUyNGEyYWM5MIIBIjANBgkqhkiG9w0B
                AQEFAAOCAQ8AMIIBCgKCAQEA0QSrUl8+53E7jbC/RjIlw6NfQSpCReg2RqpFxESl
                h/lFEF0AYaGmUqBfPKNYs5Qn/IMH30B7YzLwNA86EKXc1Muv4Im819i9vw8TkNmH
                c8RJ2+SLR/Fn8GO4UwbM+S4EWZWFjnIzgnvSCrLdGOzu6MJpLKsiof1ZMX1mQKkT
                B5S7NYzYMfEjoBPwm0HW0LT91Bx7m4KlvSLF/myzCpRt57wgL6dhPPRlO0WsblcK
                OVgPirIXJnHDRIjge9QxjTLQFq02tEdTxi1EzLHzFvdch0sBmfn4hKkoxVmM0uMi
                kxEDF67U0UcjiRtJOdDINLf32j2o5JVFder9mvDTIQJMTwIDAQABoyMwITAOBgNV
                HQ8BAf8EBAMCAgQwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEA
                kAPJLWQpDvL7O0in+xFPLVEZ6DBV2RbnbOyI+VmePNAQF2zLWf+cDBfa0r9zwhvo
                bt29mqcRdcekfD0gm2l3oWzkVmxxw/aa24uFU227SWAxbLMkKSU3ZuEsC91H3wgZ
                CFtMOUPgQhiyVJzOyR8GCRVDdUmlGSSeH/FYZUsaoeQixV2cNRWy1F0gJbji3n5g
                HTEjE6wAUem5A8gvlq8sC4PZHm+voObHwZqQT1zITKSGc9l1H39sePJHby16li4H
                NdY6bX4ARJEta1rXir15STzJ3QNioVxb7Jb5OYQdW9x7ZBZkZlr4KEwJm4+BCR4j
                8DcWLJUg1uIQ0EpwUcJd3Q==
                -----END CERTIFICATE-----
            </ca-certificate>
            <kubernetes-master>https://35.237.203.126</kubernetes-master>
        </kubernetes>
    </network>
</hazelcast-client>
```